plugins/versionpress/src/Database/wordpress-schema.yml

```diff
@@ -13,6 +13,8 @@ post:
     #  - Attachments without post_title
     #  - Updating draft via AJAX
     #  - Any changes saved by clicking "Preview"
+  ignored-columns:
+    - comment_count: '@\VersionPress\Synchronizers\PostsSynchronizer::fixCommentCounts'
```

Comment:
It is both simple and slightly confusing :) Can't think of a better alternative though, let's stick with the current format.